src/pages/legal/tos.jsx

```diff
@@ -0,0 +1,110 @@
+var React = require('react/addons');
+var Footer = require('../../components/footer/footer.jsx');
+
+module.exports = React.createClass({
+  mixins: [
+    React.addons.LinkedStateMixin
+  ],
+  render: function () {
+    return (
+      <div id="splash">
+        <div id="masthead">
+          <div className="inner">
+            <div id="header">
+              <a href="#/"><img src="./img/logo.png" width="200" height="53" alt="Webmaker" /></a>
+            </div>
+          </div>
+        </div>
+
+        <div id="mid">
+          <div className="inner legal">
+            <h1>Webmaker & teach.mozilla.org Terms of Use</h1>
```

Comment:
that `&` looks tricky. I know React can do this, but does this have to be an ampersand instead of the plain word "and"?